internal/discovery/authData.go

```diff
@@ -59,9 +77,10 @@ func getReachable(ips []net.IP, port int) (*net.IP, error) {
 	}
 }
 
-func isReachable(address string, port int) bool {
-	//_, err := net.DialTimeout("tcp", fmt.Sprintf("%s:%d", address, port), 500*time.Millisecond)
-	//return err == nil
-	// TODO: no service is available yet
-	return true
+// check if this address + port is reachable with TCP
+// the service is reachable is we are able to establish a TCP connection before a
```

Comment:
before a?

Reply:
I'm sorry... It was ".. before the timeout"